exercises/03-keywords_and_variables.md

```diff
@@ -1,74 +1,83 @@
-    Input Password    id=password_field    mode
-    Click Element    id=login_button
-    Page Should Contain    Welcome Page
-    Location Should Be    http://localhost:7272/welcome.html
-    Title Should Be    Welcome Page
+    New Browser    headless=${FALSE}
+    New Page    http://localhost:7272
+    Fill Text    id=username_field    demo
+    Fill Secret    id=password_field    mode
+    Click    id=login_button
+    Get Text    body    contains    Welcome Page
+    Get Url    ==    http://localhost:7272/welcome.html
+    Get Title    ==    Welcome Page
 ```
 
 </details>
 
-This test is technically correct, but can be quite hard to read if you don't know e.g. what is HTML attribute `id`.
+This test is technically correct, but can be quite hard to read if you don't know e.g. what is an HTML
+attribute `id`.
 
 In other words, we need to refactor our test a bit.
 
 We need to add a new table `*** Keywords ***`. It can be above or below the `*** Test Cases ***`.
 
-Under `Keywords` table we need to create a new keyword called `Open Browser To Login Page`. The same indentation
-rules apply with keywords as in test cases: each unindented line is considered a new keyword, so the implementation
-of the keyword need to be indented with 4 spaces.
+Under the `Keywords` table we need to create a new keyword called `Open Browser To Login Page`. The
+same indentation rules apply with keywords as in test cases: each unindented line is considered a new
+keyword, so the implementation of the keyword need to be indented with 4 spaces.
 
-```
+```robot
 *** Keywords ***
 
 Open Browser To Login Page
-    Open Browser    http://localhost:7272   browser=ff
+    New Browser    headless=${FALSE}
+    New Page    http://localhost:7272
 ```
 
-Now, we can replace our `Open Browser    http://localhost:7272   browser=ff` from our test case with our own
-user defined keyword `Open Browser To Login Page` to provide context to our test case.
+Now, we can replace the first two lines from our test case with our own user defined keyword
+`Open Browser To Login Page` to provide context to our test case.
 
-You can test out that your refactoring was done correctly by running `robot robot/login.robot` and to see that test is still passing.
+You can test out that your refactoring was done correctly by running `robot robot/login.robot` and
+to see that test is still passing.
 
 After the possible fixes create the following keywords:
 
-  - `Enter Username`
-  - `Enter Password`
-  - `Submit Login Form`
-  - `Verify That Welcome Page Is Visible`
+- `Enter Username`
+- `Enter Password`
+- `Submit Login Form`
+- `Verify That Welcome Page Is Visible`
 
-*Pro tip 1* One user defined keyword can contain multiple keywords, just like a test case can have multiple steps. This can be used to shorten test cases by hiding long chains of actions inside aptly named keywords.
+*Pro-tip 1:* One user defined keyword can contain multiple keywords, just like a test case can have
```

Comment:
I think all tips should use the same syntax. Now there's `pro-tip` and `tip` mixed.

Reply:
Did some unifiying for the "pro tips" 😊